src/xmlutils.cpp

```diff
@@ -23,20 +23,23 @@ limitations under the License.
 namespace libcellml {
 
 /**
- * @brief Return a map of namespaces on attributes for this node.
+ * @brief Return a list of namespaces on attributes for this node.
  *
- * Scans all attributes of the node and records any associated namespace
- * attached to the attribute in an @c XmlNamespaceMap.
+ * Scans all attributes of the node and records anyassociated
```

Comment:
typo. Not picked up by codespell? or codespell hasn't run against this PR?

Reply:
Yup, spotted that one too, but yes it ran: https://github.com/cellml/libcellml/pull/1225/checks.

Reply:
[c54d8e9](https://github.com/cellml/libcellml/pull/1225/commits/c54d8e922f2f542ba75e90245e0119e01f95923d)